ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -200,13 +213,10 @@ StackLayout {
 
         Rectangle {
             id: inputArea
-            color: Style.current.background
```

Comment:
You need to set a background anyway. Probably to `transparent`, otherwise, the bg is white